### PR TITLE
feat: build cw721 poap contract

### PR DESCRIPTION
**Pull request by dadamu**
Since the issue of the original cw721-base has `Empty` as `Extension` instead of our custom `Metadata` with claimer. This PR intends to fix the issue by building our custom cw721 contract for POAP.

### CODE REVIEW

File: contracts/poap-manager/src/test_utils.rs [hunk 1]
```diff
@@ -6,6 +6,7 @@ use cw721_base::{
     ContractError as Cw721ContractError, Cw721Contract, ExecuteMsg as Cw721ExecuteMsg,
     InstantiateMsg as Cw721InstantiateMsg, QueryMsg as Cw721QueryMsg,
 };
+use cw721_poap::{Metadata};
```
Comment (manu0466) [inline contracts/poap-manager/src/test_utils.rs:9]:
```suggestion
use cw721_poap::Metadata;
```
Lint error

File: contracts/cw721-poap/rustfmt.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# stable
```
Comment (manu0466) [inline contracts/cw721-poap/rustfmt.toml:1]: Can we move this file to the project root directory so that we have a common style for each contract?

Comment (dadamu) [reply contracts/cw721-poap/rustfmt.toml]: Removed all rustfmt in the contracts.

**Review by manu0466 — APPROVED** [review]
Nice work!

**Review by bragaz — APPROVED** [review]
Looks good to me 👍

**Pull request merged**